src/app/Http/Controllers/VanController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Van;
+use Illuminate\Http\Request;
+
+class VanController extends Controller
+{ 
+    public function storeVan(Request $request)
+    {
+        $newVan = new Van();
+        $newVan->licenceplate = $request->licenceplate;
+
+        $newVan->save();
+        return redirect()->back()->with("success_van", "Van created successfully");
+    }
+
+    public function deleteVan(Request $request)
+    {
+        $vanId = $request->van_id;
+        $van = Van::find($vanId);
+
+        if(!$van) {
+            return "Van not found!";
```

Comment:
Hiervoor graag gebruik maken van deze exception:
`use Illuminate\Validation\ValidationException;`
`throw ValidationException::withMessages(['errors' => 'This value is incorrect']);`


Dit is een build in laravel exception die je redirect naar de vorige page met een error message